tests/unit/acquisition/test_rule.py

```diff
@@ -185,7 +185,7 @@ def test_trust_region_for_unsuccessful_local_to_global_trust_region_reduced() ->
     eps = 0.5 * (search_space.upper - search_space.lower) / 10
     previous_y_min = dataset.observations[0]
     is_global = False
-    acquisition_space = Box(dataset.query_points - eps, dataset.query_points + eps)
+    acquisition_space = Box(dataset.query_points[1] - eps, dataset.query_points[1] + eps)
```

Comment:
actually it should be dataset.query_points[0]

Reply:
are you certain?